projects/plugins/inspect/rollup.config.js

```diff
@@ -0,0 +1,96 @@
+import alias from '@rollup/plugin-alias';
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+import livereload from 'rollup-plugin-livereload';
+import postcss from 'rollup-plugin-postcss';
+import svelte from 'rollup-plugin-svelte';
+import { terser } from 'rollup-plugin-terser';
+import sveltePreprocess from 'svelte-preprocess';
+
+const production = ! process.env.ROLLUP_WATCH;
+const runServer = !! process.env.SERVE;
+
+// function serve() {
+// 	let server;
+
+// 	function toExit() {
+// 		if (server) server.kill(0);
+// 	}
+
+// 	return {
+// 		writeBundle() {
+// 			if (server) return;
+// 			server = require('child_process').spawn('npm', ['run', 'start', '--', '--dev'], {
+// 				stdio: ['ignore', 'inherit', 'inherit'],
+// 				shell: true
+// 			});
+
+// 			process.on('SIGTERM', toExit);
+// 			process.on('exit', toExit);
+// 		}
+// 	};
+// }
+
+export default {
+	input: 'app-ui/src/jetpack-inspect.ts',
+	output: {
+		sourcemap: true,
+		format: 'iife',
+		name: 'app',
+		file: 'app-ui/build/jetpack-inspect.js',
+	},
+	plugins: [
+		alias( {
+			entries: [
+				// If you add a new top-level-folder besides src which you want to use, add it here
+				{ find: /^@src(\/|$)/, replacement: `${ __dirname }/app-ui/src/` },
+			],
+		} ),
```

Comment:
May as well remove this since it looks like you got rid of almost all the uses of these in favor of importing the paths directly.

That would also allow removing the dep on `@rollup/plugin-alias` from package.json.